### ISSUE TITLE
Using rlang deprecated functions

### ISSUE THREAD

**Issue by dipterix** [tlverse/delayed#18]
```r
> delayed_norm <- delayed(rnorm(n = const))
Warning message:
`lang_tail()` is deprecated as of rlang 0.2.0.
This warning is displayed once per session. 
```

Also:
```r
> chained_norm_pois$compute(nworkers = 2, verbose = TRUE)
run:0 ready:2 workers:2
updating rnorm(n = const) from ready to running
run:1 ready:1 workers:2
updating rpois(n = const, lambda = const) from ready to running
run:2 ready:0 workers:2
updating rpois(n = const, lambda = const) from running to resolved
updating rnorm(n = const) from running to resolved
updating mapfun(x = delayed_norm, y = delayed_pois) from waiting to ready
run:0 ready:1 workers:2
updating mapfun(x = delayed_norm, y = delayed_pois) from ready to running
run:1 ready:0 workers:2
updating mapfun(x = delayed_norm, y = delayed_pois) from running to resolved
[1] -0.9704017 -0.9228922 -0.9933805 -1.1188000 -1.1814955 -0.8315948 -1.1136447
Warning messages:
1: `mut_node_cdr()` is deprecated as of rlang 0.2.0.
This warning is displayed once per session. 
2: [ONE-TIME WARNING] Forked processing ('multicore') is disabled in future (>= 1.13.0) when running R from RStudio, because it is considered unstable. Because of this, plan("multicore") will fall back to plan("sequential"), and plan("multiprocess") will fall back to plan("multisession") - not plan("multicore") as in the past. For more details, how to control forked processing or not, and how to silence this warning in future R sessions, see ?future::supportsMulticore 
```

**Comment by nhejazi** [comment]
Thanks for reporting this. As per https://github.com/r-lib/rlang/blob/master/NEWS.md, we've replaced `mut_node_cdr` with `node_poke_cdr` in a9be725; however, `rlang` NEWS indicates that `lang_tail` and related functions have been soft-deprecated without replacement. We'll look into resolving that soon.

**Comment by nhejazi** [comment]
a50c5bb replaces `lang_tail` with `call_args` and `lang_fn` with `call_fn` in order to avoid the soft-deprecated `lang_*` functions.

**Comment by nhejazi** [comment]
Resolved by #17.
